services/app-api/src/domain-models/UserType.d.ts

```diff
@@ -1,19 +1,34 @@
 // CognitoUserType is our type for representing the information we get
 // from Cognito about a given user
 
-type UserType = StateUserType | CMSUserType
+type UserType = StateUserType | CMSUserType | AdminUserType
 
 type StateUserType = {
     role: 'STATE_USER'
     email: string
     name: string
     state_code: string
+    givenName: string
+    familyName: string
+    euaID: string
```

Comment:
State Users definitely dont’ have an EUA ID. I think if we don’t have anything we’re planning to use the EUA id for rn and it’s complicating this work then it makes sense to me to drop it. I’m trying to think of why we would use someone’s EUA down the road, maybe we’d need it to communicate who did some action to a third party system?

Reply:
Iagree, I think we can drop EUA. We could also use email address to communicate who did an action

Reply:
euaID has been dropped from everywhere